webapp/packages/plugin-top-app-bar/src/TopNavBar/AppStateMenu/AppStateMenu.tsx

```diff
@@ -46,7 +46,9 @@ export const AppStateMenu = observer(function AppStateMenu() {
   return (
     <SContext registry={registry}>
       <div className={s(styles, { menuWrapper: true, appStateMenu: true })}>
-        <MenuBar menu={menu} nestedMenuSettings={{ modal: true }} />
+        <Loader className={s(styles, { loader: true }, 'secondary')} secondary suspense small inline>
```

Comment:
Why do we duplicate secondary as class and secondary as prop here?

Reply:
cuz it can be another loader in MenuBar to display is also as secondary